### PR TITLE
Added sensible extensions for dealing with query strings

### PR DESCRIPTION
**Pull request by JoeStead**
Added extension methods for working with query strings.

### CODE REVIEW

File: test/BotwinModuleTests.cs [hunk 1]
```diff
@@ -174,5 +174,28 @@ public BotwinModuleTests()
...
+            var response = await this.httpClient.GetAsync($"/querystring?id={idToTest}");
+
+            var body = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal(200, (int)response.StatusCode);
+            Assert.True(body.Contains($"Managed to parse an int {idToTest}"));
+        }
+        
+        [Fact]
+        public async Task Should_return_GET_requests_with_multiple_parsed_querystring()
```
Comment (jchannon) [inline test/BotwinModuleTests.cs]: Worth a test if you do `?id=1,2,3`?

Comment (JoeStead) [reply test/BotwinModuleTests.cs]: Not tested, but I imagine it would fail on the parsing, because the query collection would return that as one value.

Comment (jchannon) [reply test/BotwinModuleTests.cs]: not sure it would because query[id] returns ienumerable<string> so i'd expect that with 1,2,3 in it

Comment (JoeStead) [reply test/BotwinModuleTests.cs]: That returns one value. I don't think we should go down the route of splitting that manually either

Comment (JoeStead) [reply test/BotwinModuleTests.cs]: Thinking about it, another overload to handle that would be quite nice.

**Pull request merged**